src/common/cuepoint/cuepoint-event-type.js

```diff
@@ -0,0 +1,22 @@
+// @flow
+const namespace = 'kaltura-player';
+
+/**
+ * @const {Object} CuePointEventType
+ *
+ * @example
+ * // How to use
+ * player.addEventListener(KalturaPlayer.cuepoint.Event.TIMED_METADATA_ADDED, e => {
+ *   console.log(e.payload);
+ * };
+ */
+const CuePointEventType: {[event: string]: string} = {
+  /**
+   * Fires when new cue-points added to text-track.
+   * @event TIMED_METADATA_ADDED
+   * @memberof CuePointEventType
+   */
+  TIMED_METADATA_ADDED: `${namespace}-timedmetadataadded`
```

Comment:
maybe add here also the `TIMED_METADATA`, (the fact it comes from the player it's internal detail, it will be clearer if both events are coming from cue point). but if so need to remove the `kaltura-player` namespace